packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -362,13 +360,19 @@ class ClayMultiSelect extends ClayComponent {
 				};
 			})[0];
 
-		item = filteredItem || item;
```

Comment:
hey @carloslancha I was thinking today, it would be interesting to add some comments which use case the `filteredItem` covers, I took a while to understand this, probably someone else will also take a while. Just to leave the reasons why you added this.

Reply:
I think now is clearer. You just need to read JSDOCS form creatable param